rootfs/scheduler/__init__.py

```diff
@@ -652,7 +652,8 @@ def _create_rc(self, name, image, command, **kwargs):  # noqa
             containers[0]["resources"]["limits"]["cpu"] = cpu
 
         # add in healtchecks
-        js_template = self._healthcheck(js_template, **kwargs['healthcheck'])
+        if kwargs.get('healthcheck', {}):
```

Comment:
don't we want this to be just `if kwargs.get('healthcheck'):` so it'll fail if the `healthcheck` kwarg doesn't exist?